packages/zoe/src/zoeService/offer/offer.js

```diff
@@ -55,9 +55,9 @@ export const makeOfferMethod = (
     instanceAdmin.assertAcceptingOffers();
 
     const proposal = cleanProposal(uncleanProposal, getAssetKindByBrand);
-    const proposalSchema = getProposalSchemaForInvitation(invitationHandle);
-    if (proposalSchema !== undefined) {
-      fit(proposal, proposalSchema, 'proposal');
+    const proposalShape = getProposalSchemaForInvitation(invitationHandle);
```

Comment:
should this also be `getProposalShape`?

Reply:
Good catch! Doing a deeper search now.

Reply:
Done. Optimistically resolving, but PTAL